docs/v4.0/layers/point-density-grid-layer.md

```diff
@@ -0,0 +1,63 @@
+<p align="right">
+  <img src="https://img.shields.io/badge/extruded-yes-blue.svg?style=flat-square" alt="64-bit" />
+</p>
+
+# PointDensityGridLayer
+
+The PointDensityGridLayer renders a grid heatmap based on an array of points.
+It takes the constant size all each cell, projects points into cells. The color 
+and height of the cell is scaled by number of points it contains.
```

Comment:
At the moment my understanding is that this layer only works with `COORDINATE_SYSTEM.LNG_LAT` so we need to make this restriction quite clear here.